vuepress/docs/next/docs/reference/bundle-upgrade.md

```diff
@@ -0,0 +1,141 @@
+
+The improvements include:
+* A simplified bundle directory structure
+* A decoupled build and deployment process that is easier to understand, modify, and troubleshoot
+* CI/CD integration and support 
+* A single required Git source repository with Docker image repositories created automatically as needed
+* Optimized file storage and transfer methods using a layered file system, beneficial for large projects
+* A standardized bundle address format
+
+## Quick Reference
+* [Bundle Specifications](../curate/bundle-details.md)
+
+* [Bundle Management](../getting-started/ent-bundle.md)
+
+* [Bundle Version Comparison Table](./bundle-comparison.md)
+
+## Upgrade Process
+
+### 1. Initialize Bundle 
+
+Initialize a new bundle to set up the docker-based bundle structure using the ent CLI:
+
+``` 
+ent bundle init YOUR-BUNDLE-NAME
+```
+### 2. Copy Assets
+1. Use the ent CLI bundler command to export assets from an existing Entando Bundle. See the step-by-step [Export and Publish Tutorial](../../tutorials/create/pb/export-bundle-from-application.md). 
+
+2. The `platform` directory is designated for resources such as fragments, contents, pages, and templates. Copy the output of the ent bundler to 'YOUR-BUNDLE-NAME/platform`. 
+      
+>-You can also accomplish this manually, but the ent bundler can automatically generate top-level descriptors for platform resources.  
+-Widgets that are not MFEs are considered platform entities on Entando and should be placed in the `platform/widgets` directory.
+
+### 3. Add Micro Frontends & Microservices
+ 
+1. From inside your bundle project folder, add your micro frontend to the bundle descriptor `entando.json`:
+``` sh
+ent bundle mfe add YOUR-MFE1
+```
+>Docker-based bundles have a centralized bundle descriptor that includes specifications for bundle entities such as micro frontends, microservices, APIs, DBMS, and ID management. 
+
+2. Copy the MFE files into the `YOUR-BUNDLE-NAME/microfrontends` directory. For instance, for an MFE named YOUR-MFE1: 
+
+``` sh
+mv YOUR-ORIGIN-DIRECTORY/YOUR-MFE1/{.,}* microfrontends/YOUR-MFE1
+```
+>Micro frontends and microservices are partitioned into their own directories in v5 bundles to streamline the CI/CD process.
+
+3. Add your microservice:
+``` sh
+ent bundle ms add YOUR-MS1
+```
+4. Copy YOUR-MS1 files to the new location:
+``` sh
+mv YOUR-ORIGIN-DIRECTORY/YOUR-MS1/{.,}* microservicess/YOUR-MS1
+```
+5. Repeat steps 1 through 4 for all micro frontends and microservices.
+
+### 4. Add API Claims
+1. Add an API claim to connect `YOUR-MFE1` to a microsersive as needed. The connection information is added to the descriptor `entando.json`.
+```shell
+ent bundle api add YOUR-MFE1 YOUR-API --serviceName=YOUR-MS1 --serviceUrl=http://localhost:8081
+```
+
+2. Repeat the previous step for all APIs as needed.
+
+>Refer to the [API Management](../getting-started/ent-api.md) page for additional information.
+
+### 5. Add Custom Commands to Test Locally
```

Comment:
This is an optional step. Even without a custom command you can run the MFEs/MS as-is, you just cannot run them at the same time they will collide. I might just add (Optional) after 5.